includes/wizards/class-popups-wizard.php

```diff
@@ -523,7 +537,7 @@ function( $prompt ) {
 					$prompt['edit_link'] = get_edit_post_link( $prompt['id'] );
 					return $prompt;
 				},
-				$newspack_popups_configuration_manager->get_prompts( true )
+				$newspack_popups_configuration_manager->get_prompts( true, true )
```

Comment:
Non-blocking nit-picking: maybe it's time this becomes a configuration object?

Reply:
I've found it cumbersome in PHP to use this pattern - too much boilerplatey checking for key existence etc. But feel free to propose a change - it's likely I'm not thinking about the most elegant implementation of it.